src/Sylius/Bundle/ApiBundle/Converter/CommandFieldItemIriToIdentifierStorage.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Converter;
+
+final class CommandFieldItemIriToIdentifierStorage implements CommandFieldItemIriToIdentifierStorageInterface
+{
+    /** @var string */
+    private $commandType;
```

Comment:
```suggestion
    private $commandClass;
```

It's a FQCN that is stored there.